translations/de.ts

```diff
@@ -52,26 +43,20 @@
         <translation>Aufnahmegerät</translation>
     </message>
     <message>
-        <source>Filter audio</source>
-        <translation>Aufnahme filtern</translation>
-    </message>
-    <message>
-        <source>0</source>
-        <translatorcomment>Ausgelassen</translatorcomment>
-        <translation></translation>
+        <source>Use slider to set volume of your microphone.</source>
+        <translation>Verwende den Schieber, um die Aufnahmelautstärke deines Mikrofons anzupassen.</translation>
     </message>
     <message>
-        <source>Filter sound from your microphone, so that people hearing you would get better sound.</source>
-        <translation>Das Filtern kann die Tonqualität verbessern.</translation>
+        <source>Gain</source>
+        <translation>Lautstärke</translation>
     </message>
     <message>
-        <source>100</source>
-        <translatorcomment>Ausgelassen</translatorcomment>
+        <source>...</source>
         <translation></translation>
```

Comment:
It doesn't matter because if there's no translation the original string will be displayed, what's `...` as well. ;)

Anyway, this string/translation is useless in general.